widgets/qp_canvas.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * This class defines a canvas and how it will draw its components
+ */
+class QPCanvas
+{
+    handle = undefined;
+    width = 0;// CONSTANTS
```

Comment:
Can you fix the spacing of the comments on here and line 9?

Reply:
I'm not even sure why that was there in the first place lmao.